evap/staff/forms.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, *args, **kwargs):
         self.fields["evaluation"].choices = choices
 
     def clean(self):
-        if self.evaluation_selection_required and self.cleaned_data["evaluation"] is None:
+        if self.evaluation_selection_required and self.cleaned_data.get("evaluation") is None:
```

Comment:
just stumbled on this bug: If there was no `evaluation` in `self.cleaned_data` we'd just get a `ValueError` instead of a `ValidationError`